dist/index.css

```diff
@@ -384,3 +385,96 @@ main {
   .btn-link:focus, .btn-link.focus {
     text-decoration: underline;
     box-shadow: none; }
+
+.ui.input {
```

Comment:
do not put your css in this file directly. check how I did for button. 
And do not use .ui.input selecotor